### DISCUSSION TITLE
If condition in form builder

### DISCUSSION THREAD

**Discussion by paulustimothy** [filamentphp/filament#12541] [Help] [answered]
### Package

Form builder

### Package Version

3.2.71

### How can we help you?

Select::make('payment_method')
                            ->options([
                                'stripe' => 'Stripe',
                                'cod' => 'Cash On Delivery'
                            ])
                            ->native(false)
                            ->required(),

Select::make('shipping_method')
                            ->options([
                                'j&t' => 'J&T',
                                'jne' => 'JNE'
                            ])
                            ->native(false),

how to make if payment_method is 'cod' shipping_method is disabled
thanks

**Comment by leandrocfe** [comment, marked as answer]
```php
Select::make('payment_method')
->options([
'stripe' => 'Stripe',
'cod' => 'Cash On Delivery'
])
->native(false)
->required()
->live(),

Select::make('shipping_method')
->options([
'j&t' => 'J&T',
'jne' => 'JNE'
])
->native(false)
->disabled(fn (Forms\Get $get): bool => $get('payment_method') === 'cod')
```
